provisioning/resources/elasticsearch/mapping/good-mapping.json

```diff
@@ -248,79 +240,79 @@
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "refr_urlpath": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "refr_urlport": {
                     "type": "long"
                 },
                 "refr_urlquery": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "refr_urlscheme": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "se_action": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "se_category": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "se_label": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "user_fingerprint": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "user_id": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "user_ipaddress": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "useragent": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "v_collector": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "v_etl": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
                 },
                 "v_tracker": {
-                    "type": "string",
-                    "index": "not_analyzed"
+                    "type": "keyword",
+                    "index": true
```

Comment:
what are the implications of those changes? cc @jbeemster

Reply:
`"string"` field is [unsupported](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/string.html) in favor of `text` and `keyword`. I followed Elastic's official [recommendations](https://www.elastic.co/blog/strings-are-dead-long-live-strings).